Sources/PointFree/TODO.swift

```diff
@@ -16,6 +16,27 @@ public func array<A>(_ tuple: (A, A, A, A, A, A, A, A, A)) -> [A] {
   return [tuple.0, tuple.1, tuple.2, tuple.3, tuple.4, tuple.5, tuple.6, tuple.7, tuple.8]
 }
 
+func filterMapValues<Key, Value, NewValue>(
```

Comment:
Added some `filterMap` helpers for dictionary values. Would be nice to get `{filter,partition}Map`s of all kinds into Prelude...